src/components/index.ts

```diff
@@ -0,0 +1 @@
+export { default as Axis } from './axis';
```

Comment:
Should we discuss named export vs default?

Reply:
sure